app/controllers/schools/dashboard_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Schools::DashboardController < Schools::BaseController
+  skip_after_action :verify_authorized
+  skip_after_action :verify_policy_scoped
```

Comment:
Are we ever using those in a controller inheriting from `Schools::BaseController`? :D

Reply:
Ah but we will be!